WordPress/src/test/java/org/wordpress/android/ui/posts/EditPostRepositoryTest.kt

```diff
@@ -499,6 +553,40 @@ class EditPostRepositoryTest {
         assertThat(editPostRepository.getPost()).isEqualTo(post)
     }
 
+    @Test
+    fun `loads parentTitle from store for post loaded by remote id is not 0`() = test {
+        val remoteId = 2L
+        val parentId = 10L
+
+        val post = PostModel().apply { setParentId(parentId) }
+        val site = SiteModel()
+
+        val parentTitle = "Parent"
+        val parentPost = PostModel().apply { setTitle(parentTitle) }
+
+        whenever(postStore.getPostByRemotePostId(remoteId, site)).thenReturn(post)
+        whenever(postStore.getPostByRemotePostId(parentId, site)).thenReturn(parentPost)
+
+        editPostRepository.loadPostByRemotePostId(remoteId, site)
+
+        assertThat(editPostRepository.parentTitle).isEqualTo(parentTitle)
+    }
+
+    @Test
+    fun `loads parentTitle from store for post loaded by remote id is 0`() = test {
```

Comment:
The name of this test can be improved imho, I find the title of [this test similar to it](https://github.com/wordpress-mobile/WordPress-Android/pull/17329/files#diff-edb47136a3f8bef313720de26c2e70f0a4ba92239884f7dc0545a4b9a4f08df8R530) a bit cleaner.

Reply:
Done!